frontend/components/site/SitePublishedFilesTable.jsx

```diff
@@ -1,41 +1,141 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import autoBind from 'react-autobind';
 
 import publishedFileActions from '../../actions/publishedFileActions';
 import LoadingIndicator from '../LoadingIndicator';
 import AlertBanner from '../alertBanner';
 
 class SitePublishedFilesTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentPage: 0,
+      filesByPage: {},
+      lastPage: null,
+    };
+  }
+
   componentDidMount() {
     const site = { id: this.props.params.id };
     const branch = this.props.params.name;
-    publishedFileActions.fetchPublishedFiles(site, branch);
+
+    const startAtKey = null; // start without a startAtKey
+    publishedFileActions.fetchPublishedFiles(site, branch, startAtKey);
+
+    autoBind(this, 'previousPage', 'nextPage', 'shouldDisablePreviousPage',
```

Comment:
Is `autoBind` supposed to be called here? Generally binding like this happens in the constructor. Just wondering!

Reply:
you're right - bad copy-paste, thanks!